plancton/__init__.py

```diff
@@ -508,17 +514,18 @@ def main_loop(self):
       except Exception as e:
         self.logctl.error("Cannot pull Docker image %s: no new containers, will retry later" % \
                           self.conf["docker_image"])
-    if prev_influxdb_url != self.conf["influxdb_url"]:
+    if set(prev_influxdb_url).symmetric_difference(set(self.conf["influxdb_url"])):
```

Comment:
This is good to know, great! My comment on making `self.conf["influxdb_url"]` a set still stands, so that you would save two `set()` conversions on this line.